styles/bsky/catppuccin.user.css

```diff
@@ -2,7 +2,7 @@
 @name Bluesky Social Catppuccin
 @namespace github.com/catppuccin/userstyles/styles/bsky
 @homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/bsky
-@version 0.0.9
```

Comment:
I'd prefer if this was changed to 0.1.0 instead of 0.0.10

Reply:
i have no idea how versioning is supposed to work in userstyles tbh 🙃